File: Olympiad/COCI/official/2007/contest2_solutions/r2.pas

(*
  Hrvatsko otvoreno natjecanje iz informatike 2006/2007
  2. kolo - Zadatak R2
  Programski jezik Pascal

  Po definiciji aritmeticke sredine vrijedi s=(r1+r2)/2. Iz te
  jednadzbe slijedi r2=2*s-r1.
*)

program r2;

var r1, s : integer;

begin
   readln(r1, s);
   writeln(2*s - r1);
end.

File: Olympiad/COCI/official/2007/contest2_solutions/r2.c

/*
  Hrvatsko otvoreno natjecanje iz informatike 2006/2007
  2. kolo - Zadatak R2
  Programski jezik C

  Po definiciji aritmeticke sredine vrijedi s=(r1+r2)/2. Iz te
  jednadzbe slijedi r2=2*s-r1.
*/

#include <stdio.h>

int main(void)
{
   int r1, s;
   scanf("%d%d", &r1, &s);
   printf("%d\n", 2*s-r1);

   return 0;
}
